### PR TITLE
Reorganize newly added fields in scholarx.json and add content

### LINKED ISSUE #522: Reorganize newly added fields in scholarx.json
**Is your feature request related to a problem? Please describe.**
- Social links are not properly organized right now.

**Describe the solution you'd like**
- Social links have to go under a new key and it should be an array.
"socialLinks" : [
   {
    "platform" : "Linkedin",
    "icon" : "fa-linkedin",
     "url" : "..."
   },
    {
    "platform" : "Facebook",
    "icon" : "fa-linkedin",
     "url" : "..."
   },
]
- Same with the project. The key should be "projects" and it should be an array.
- The keys should not contain white spaces and must use simpleCamel case format.
- Add this content from this link https://docs.google.com/spreadsheets/d/1zOEfTBTTBDdDKcafHI44CY73v2obLEnnOupNBzzO0Do/edit?usp=sharing

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #522 

## Goals

- Reorganize newly added fields in scholarx.json

## Approach

- Reorganize newly added fields in scholarx.json
- Add content

### Screenshots

Screenshot from 2020-02-26 21-35-44

  
### Preview Link

https://pr-537-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
- Do not use spaces in the keys
- Use simple camel case
- Line length should be limited to 120 charactors. Don't go beyond the white line on the editor.

File: src/main/webapp/scholarx/2020/mentors.json [hunk 1]
```diff
@@ -1,14 +1,425 @@
...
+    "socialLinks" : [
+      {
+        "platform" : "Linkedin",
+        "icon" : "fa-linkedin",
+        "url" : "https://www.linkedin.com/in/dinithisumanaweera/"
+      },
+      {
+        "platform" : "Facebook",
+        "icon" : "fa-linkedin",
+        "url" : "..."
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx/2020/mentors.json]: Replace `...` with `#`

File: src/main/webapp/scholarx/2020/mentors.json [hunk 2]
```diff
@@ -1,14 +1,425 @@
...
+    "image": "ganindu.jpeg",
+    "bio": "Ganindu Nanayakkara is currently obtaining a Master of Business Adminstration at the University of Kelaniya. He also has a BSc in Information Technology from Curtin University and was a former researcher at NASA Ames Research Center. He can provide mentorship and assistance with topics relating to innovation, IT, business and finance.",
+    "Areas of expertise" : [
+      "Innovation",
+      "Information technology",
+      "Business",
+      "Finance"
+    ],
+    "Projects" : [
+    ],
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx/2020/mentors.json]: `"projects": [], `    do not split lines for empty arrays

File: src/main/webapp/scholarx/2020/mentors.json [hunk 3]
```diff
@@ -1,14 +1,425 @@
...
+      },
+      {
+        "platform" : "Facebook",
+        "icon" : "fa-linkedin",
+        "url" : "..."
+      }
+    ]
   }
-]
+]
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx/2020/mentors.json]: EOF

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

File: src/main/webapp/scholarx/2020/mentors.json (hunk 1 above)
Comment (anjula-sack) [reply src/main/webapp/scholarx/2020/mentors.json]: Ok

**Comment by jayasanka-sack** [comment]
> * Do not use spaces in the keys
> * Use simple camel case
> * Line length should be limited to 120 charactors. Don't go beyond the white line on the editor.

Do this one too

File: src/main/webapp/scholarx/2020/mentors.json (hunk 3 above)
Comment (anjula-sack) [reply src/main/webapp/scholarx/2020/mentors.json]: Ok

**Comment by anjula-sack** [comment]
> > * Do not use spaces in the keys
> > * Use simple camel case
> > * Line length should be limited to 120 charactors. Don't go beyond the white line on the editor.
> 
> Do this one too

I replaced AreasOfExpertise like this

**Comment by jayasanka-sack** [comment]
> > > * Do not use spaces in the keys
> > > * Use simple camel case
> > > * Line length should be limited to 120 charactors. Don't go beyond the white line on the editor.
> > 
> > 
> > Do this one too
> 
> I replaced AreasOfExpertise like this

- [ ] Use simple camel case
- [ ] Line length should be limited to 120 charactors. Don't go beyond the white line on the editor.

**Comment by anjula-sack** [comment]
> > > > * Do not use spaces in the keys
> > > > * Use simple camel case
> > > > * Line length should be limited to 120 charactors. Don't go beyond the white line on the editor.
> > > 
> > > 
> > > Do this one too
> > 
> > 
> > I replaced AreasOfExpertise like this
> 
> [ ] Use simple camel case
> [ ] Line length should be limited to 120 charactors. Don't go beyond the white line on the editor.

I can't ENTER to a new line

**Comment by jayasanka-sack** [comment]
> > > > > * Do not use spaces in the keys
> > > > > * Use simple camel case
> > > > > * Line length should be limited to 120 charactors. Don't go beyond the white line on the editor.
> > > > 
> > > > 
> > > > Do this one too
> > > 
> > > 
> > > I replaced AreasOfExpertise like this
> > 
> > 
> > [ ] Use simple camel case
> > [ ] Line length should be limited to 120 charactors. Don't go beyond the white line on the editor.
> 
> I can't ENTER to a new line

Screenshot 2020-02-27 at 13 40 37

**Comment by jayasanka-sack** [comment]
https://stackoverflow.com/questions/3837959/how-to-have-the-formatter-wrap-code-with-intellij/29922077

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
https://stackoverflow.com/questions/3837959/how-to-have-the-formatter-wrap-code-with-intellij/29922077

**Comment by jayasanka-sack** [comment]
Or just hit enter at the margins

**Comment by jayasanka-sack** [comment]
- [x] Use simple camel case
- [x] Line length should be limited to 120 charactors. Don't go beyond the white line on the editor.

**Comment by anjula-sack** [comment]
Screenshot from 2020-02-27 13-47-14
its wrapped

**Comment by jayasanka-sack** [comment]
> Screenshot from 2020-02-27 13-47-14
> its wrapped

- [ ] Use simple camel case for keys
- [ ] Line length should be limited to 120 charactors. Don't go beyond the white line on the editor.


Screenshot 2020-02-27 at 13 50 32

**Review by jayasanka-sack — APPROVED** [review]
Good job

**Pull request merged**